docker-compose.yml

```diff
@@ -5,15 +5,20 @@ services:
   nginx:
     image: nginx:1.17-alpine
     volumes:
-      - ./etc/dev/nginx/nginx.conf:/etc/nginx/conf.d/default.conf
-      - .:/app
+      - ./etc/dev/nginx/nginx.conf:/etc/nginx/conf.d/default.conf:ro
+      - .:/app:ro
```

Comment:
Why don't read only public folder ?

Reply:
That's a really good point, nginx doesn't need anything else. Will change it. Thanks for the feedback!